build/podman/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+quarkus.container-image.group=qe
+# DON'T remove the spaces that are at the end of quarkus app name
+quarkus.application.name=hello-world-app      
+quarkus.container-image.tag=1.0.0
+
+# native debug configuration
+quarkus.native.debug.enabled=true
+quarkus.native.additional-build-args=-H:GenerateDebugInfo=1, -H:-SpawnIsolates, -H:DebugInfoSourceCacheRoot=test
```

Comment:
I presume this has a price as more things will be done, are you positive we will need it and we cannot just enable it when there is a problem?